packages/kiwi-core/src/test-utils/test-utils.js

```diff
@@ -0,0 +1,39 @@
+import '@testing-library/jest-dom/extend-expect'
```

Comment:
I think these can be moved to their own `tests` directory in the package root. that way the `src` directory only keeps the components.

Reply:
moved ^^